app/models/schema-block.ts

```diff
@@ -11,6 +14,40 @@ export default class SchemaBlockModel extends OsfModel implements SchemaBlock {
     @attr('string') exampleText?: string;
     @attr('boolean') required?: boolean;
     @attr('number') index?: number;
+
+    @belongsTo('registration-schema', { inverse: 'schemaBlocks', async: false })
+    schema?: RegistrationSchemaModel;
+
+    @computed('id')
+    get elementId() {
+        return this.id;
```

Comment:
In the overview page's nav dropdown, we use `utils/slugify` -- would be nice to do something similar, for a human-readable url fragment.

Reply:
I didn't want to use the display text for the element id because the display text could be very long. In case of a paragraph block, the slug will be the entire paragraph. Also, there is no guarantee that the displayText is unique. I believe the block id is a better identifier.